charts/quickwit/values.yaml

```diff
@@ -411,7 +487,7 @@ seed:
     #       type: limit_merge
     #       max_merge_ops: 3
     #       merge_factor: 10
-    #       max_merge_factor: 12
+  #       max_merge_factor: 12
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed